packages/teleport/src/DesktopSession/ActionMenu.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { MenuIcon, MenuItem, MenuItemIcon } from 'shared/components/MenuAction';
+import * as Icons from 'design/Icon';
+import { Flex } from 'design';
+import { useTheme } from 'styled-components';
+
+export default function ActionMenu(props: Props) {
+  const theme = useTheme();
+
+  return (
+    <Flex alignItems="center">
+      <MenuIcon
+        buttonIconProps={{
+          ml: 4,
+          size: 0,
+          style: { fontSize: '20px', color: theme.colors.text.secondary },
+        }}
```

Comment:
Confused by a few things here:

- Why can I pass `color` as an object field, but need to pass `fontSize` in the `style` object?
- How/where does the string `'text.secondary'` get interpreted to mean `props.theme.colors.text.secondary`? Or is that a misunderstanding of what the `StyledThemeProvider` is passing down to child objects?

Reply:
> Why can I pass color as an object field, but need to pass fontSize in the style object?

Because of styled systems, if you dig into `MenuIcon` you'll see this: https://github.com/gravitational/webapps/blob/master/packages/design/src/ButtonIcon/ButtonIcon.jsx#L93, I think in order for `fontSize` to be recognized as a prop, we need to also include `{typography}`: https://styled-system.com/api/#typography

> How/where does the string 'text.secondary' get interpreted to mean props.theme.colors.text.secondary? Or is that a misunderstanding of what the StyledThemeProvider is passing down to child objects?

From my understanding, the styled-components `ThemeProvider` allows styled components to access the theme object: https://styled-components.com/docs/advanced#theming. If you dig into `MenuIcon`, you'll see we pass these styling props to a styled component: https://github.com/gravitational/webapps/blob/master/packages/design/src/ButtonIcon/ButtonIcon.jsx#L61